packages/broker/test/unit/ConfigWizard.test.ts

```diff
@@ -8,6 +8,19 @@ const mockPromptMethod = (wizard: ConfigWizard, mockedPromptResult: any) => {
 }
 describe('ConfigWizard', () => {
     let wizard: ConfigWizard
+    const tmpDataDir = '/tmp/streamr-broker-test/'
+    
+    // create a clean dir for the tests on every run
+    beforeAll(() => {
+        if (existsSync(tmpDataDir)) {
```

Comment:
Seems slightly hazardous to be deleting a potentially pre-existing folder here. You might want to look at using `mkdtempSync` see: https://blog.mastykarz.nl/create-temp-directory-app-node-js/ that way you can generate a unique temporary directory so you don't have to worry about it pre-existing and only leave cleanup to `afterAll`.

Reply:
Interrupted tests were making a mess, this is a much more elegant solution